### PR TITLE
UX Flow proposal

### PR DESCRIPTION
**Pull request by albertomota-codes**

### CODE REVIEW

**Comment by albertomota-codes** [comment]
@uBhappy2  I addressed your comments now it looks like this. pretty similar to you know what app.
ezgif com-video-to-gif

**Comment by albertomota-codes** [comment]
@uBhappy2  i did not get this part "we can use full screen real-estate and move it to the top".
Do you mean remove the navigation view bar?

**Comment by uBhappy2** [comment]
Thanks a lot! Will approve.

Best,
Amit

File: Ventivader/Ventivader/Managers/VentilatorParametersManager.swift [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        self.headerHighlight = headerHighlight
+    }
+}
+
+final class VentilatorParametersManager {
+    static let shared = VentilatorParametersManager()
+
+    var ventilatorParameters: [VentilatorParameterModel] = [
+        VentilatorParameterModel(title: NSLocalizedString("CONTROL MODE", comment: "Control Panel tile title"),
+                                 value: NSLocalizedString("PC-SIMV", comment: "Control Panel tile value"),
```
Comment (uBhappy2) [inline Ventivader/Ventivader/Managers/VentilatorParametersManager.swift:40]: can you  change this to AC (Assist-Control) which is also called as VCV (Volume Controlled Ventilation). This is our default mode we support currently. Later we might be able to support PCV (Pressure Controlled Ventilation).

Comment (albertomota-codes) [reply Ventivader/Ventivader/Managers/VentilatorParametersManager.swift]: Done

**Pull request merged**